docs/actions/index.md

```diff
@@ -0,0 +1,110 @@
+# Actions
+
+> yarn add @redux-tools/actions
+
+This package is a collection of utility functions for creating [FSA-compliant](https://github.com/redux-utilities/flux-standard-action) action creators and reducers.
+
+## Usage example
+
+```js
+import {
+	makeActionCreator,
+	makeActionTypes,
+	makeBinaryActionCreator,
+	makeConstantActionCreator,
+	makeSimpleActionCreator,
+	makeReducer,
+} from '@redux-tools/actions';
+
+export const ActionTypes = makeActionTypes(
+	'duck',
+	['ADD', 'ADD_CRAZY', 'INCREMENT', 'REMOVE'],
+);
+
+export const add = makeSimpleActionCreator(ActionTypes.ADD);
+export const addCrazy = makeActionCreator(ActionTypes.CRAZY_ADD, 2, null);
```

Comment:
ADD_CRAZY (typo), also, the usage of makeActionCreator is wrong.

Reply:
> ADD_CRAZY (typo), also, the usage of makeActionCreator is wrong.
Typo - fixed, but not sure what do You mean ("the usage").